src/components/SourceCodeEditor.tsx

```diff
@@ -21,7 +21,7 @@ interface ISourceCodeEditor {
 }
 
 const SourceCodeEditor = (props: ISourceCodeEditor) => {
-  const editorRef = useRef<EditorDidMount['editor'] | null>(null);
+  const editorRef = useRef<Parameters<EditorDidMount>[0] | null>(null);
```

Comment:
What is the `[0]` for?

Reply:
To pick the first parameter of the Parameters list

Reply:
The issue is that the parameters' types of the `EditorDidmount` function are not exposed, so that's why I was using the reference, to avoid doing an internal import.

```typescript
import * as monaco from "monaco-editor/esm/vs/editor/editor.api";

declare class MonacoEditor extends React.Component<MonacoEditorProps> {
    ...
    editor?: monaco.editor.IStandaloneCodeEditor;
    ...
}
```